src/Stratis.Bitcoin.Tests/Utilities/ParallelAsyncTest.cs

```diff
@@ -29,36 +32,33 @@ public ParallelAsyncTest()
         {
             int sum = 0;
 
-            Stopwatch watch = Stopwatch.StartNew();
+            var delayTask = new Task(async () => { await Task.Delay(this.testCollection.Count * this.itemProcessingDelayMs).ConfigureAwait(false); });
```

Comment:
This is a very weird test dude.

Reply:
If apro approves it then ok, but you could perhaps take a different approach. using a ManualResetEvent one tasks blocks it the other releases that will prove degree of parallelism .

Reply:
Me does not approve this, because me thinks that stopwatch is okay here, there were many other problems that are now hopefully fixed but I haven't checked carefully, but as it is just test, I am not blocking it

Reply:
>This is a very weird test dude.

Well, yeah. 
Returned the stopwatch (but now there is a larger delay so should be ok on slow environments like travis).